.gitignore

```diff
@@ -1,9 +1,18 @@
 .DS_Store
 venv
 node_modules
+bin/*
+lib/*
+share/*
+pyvenv.cfg
+.vscode/*
 
+*varfontprep*
 *.ttx
 
 mastering/build/*
 node_modules
-*glyph_widths.json
\ No newline at end of file
+instances
+*/instances/*
+*/instances
+*glyph_widths.json
```

Comment:
Did you intend to change the .gitignore, or was this carried over from before? A few of these items, we could ignore, but I find handy to keep in the repo (`.vscode./*` settings plus the `varfontprep` folders).

Reply:
Sorry, I'll change. The `.vscode` things are all tied to your copy of vscode, so I stripped it, but will change. I really view varfontprep folders as temporary, but see that you don't, so will change.

Reply:
I saw them as temporary for awhile, but really, they are semi-temporary, because they are the only way someone else (including you) can actually build fonts from what's in the repo.